blockchain/vm/errors.go

```diff
@@ -36,6 +36,8 @@ var (
 	ErrTotalTimeLimitReached             = errors.New("reached the total execution time limit for txs in a block")
 	ErrOpcodeComputationCostLimitReached = errors.New("reached the opcode computation cost limit")
 	ErrFailedOnSetCode                   = errors.New("failed on setting code to an account")
+	ErrOutOfGas                          = errors.New("out of gas")
```

Comment:
How about using kerrors?
https://github.com/kaiachain/kaia/blob/dev/kerrors/kerrors.go#L29

Reply:
As @blukat29 pointed out, that would be a good idea.
https://github.com/kaiachain/kaia/pull/119#discussion_r1830259252